option optimize_for = LITE_RUNTIME;

message CreateRoleRequest
{
 required uint32 cardId = 1;
 required string name = 2;
}
